packages/ai-bot/main.ts

```diff
@@ -134,19 +134,30 @@ class Assistant {
       (prompt.model && !this.toolCallCapableModels.has(prompt.model))
     ) {
       return this.openai.chat.completions.stream({
-        model: prompt.model ?? DEFAULT_LLM,
+        model: this.getModel(prompt),
         messages: prompt.messages as ChatCompletionMessageParam[],
+        reasoning_effort: this.getResoningEffort(prompt),
       });
     } else {
       return this.openai.chat.completions.stream({
-        model: prompt.model ?? DEFAULT_LLM,
+        model: this.getModel(prompt),
         messages: prompt.messages as ChatCompletionMessageParam[],
         tools: prompt.tools,
         tool_choice: prompt.toolChoice,
+        reasoning_effort: this.getResoningEffort(prompt),
       });
     }
   }
 
+  getModel(prompt: PromptParts) {
+    return prompt.model ?? DEFAULT_LLM;
+  }
+
+  getResoningEffort(prompt: PromptParts) {
```

Comment:
Consider adding a comment indicating that this is temporary hard-coded logic that we will replace with model presets that including reasoning effort settings